index.html

```diff
@@ -0,0 +1,522 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio: desktop version day#1</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <!-----<link rel="stylesheet" href="css/finish.css"--------->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+
+  <div class="wrapper">
+    <header class="header-2 background-color">            
+      <div class="logo-link">
+        <a href="./index.html" class="my-logo">My Logo</a>
+      </div>
+      <nav class="navbar-desk">
+        <ul class="menu-desk">
```

Comment:
- [ ] your cards are aligned to the left in the desktop version. please fix that.
![Screenshot (387)_LI](https://user-images.githubusercontent.com/65298048/187799801-f7053315-6e06-47bf-862e-cea0218e3ecd.jpg)